README.md

```diff
@@ -1 +1,5 @@
-# listen-while-reading
\ No newline at end of file
+# listen-while-reading
+
```

Comment:
This project is a web app that searches movies along with their soundtracks. 

We used jQuery, Firebase, Bootstrap, Movie API, and Spotify API.

Team Members: Marcel Garza, Rhonda Gilliard, Scott Semotan